protected/views/admin/generic/list.php

```diff
@@ -81,11 +81,16 @@
                 <?php
                 $retrieveResults = $admin->getSearch()->retrieveResults();
                 foreach ($retrieveResults as $i => $row) { ?>
+                    <?php if ($admin->getModelName()=="LetterType" && $admin->attributeValue($row, 'name')=="Internal Referral"):?>
+                        <tr class="" data-id="<?php echo $row->id ?>" onclick="<?php Yii::app()->user->setFlash('notice', "Internal Referral Type can not be changed.");?>" >
```

Comment:
A better way to display a warning to the user might be by using a javascript popup window. OpenEyes comes with a decent API for them, but a basic example of adding one would look like this:
```
new OpenEyes.UI.Dialog.Alert({
            content: "You can't accept this participant into your Trial because that participant has already been accepted into another Intervention trial."
          }).open()
```